src/components/common/needsShutdown.tsx

```diff
@@ -232,14 +236,15 @@ export const NeedsShutdownAlert = ({ idPrefix } : { idPrefix: string }) =>
     <Alert isInline id={`${idPrefix}-idle-message`} customIcon={<PendingIcon />} title={NEEDS_SHUTDOWN_MESSAGE} />;
 
 export const VmNeedsShutdown = ({ vm } : { vm: VM }) => {
+    const Dialogs = useDialogs();
     const devices = getDevicesRequiringShutdown(vm);
 
     if (devices.length === 0)
         return;
 
-    const body = (
+    const body = (hide: () => void) => (
         <>
-            {_("Some configuration changes only take effect after a fresh boot:")}
+            {_("Some configuration changes only take effect after a complete shutdown and restart:")}
```

Comment:
Thinking aloud: string change, but makes it consistent with the rest of the UI. :+1: